src/components/DatePicker/DatePicker.jsx

```diff
@@ -51,12 +52,10 @@ export const DatePicker = ({
     });
     const [endMonth, setEndMonth] = useState(() => {
         if (!value || !value.to || !value.from) {
-            return dayjs(new Date()).add(1, "month").toDate();
+            return now(new Date()).add(1, "month").toDate();
```

Comment:
```suggestion
            return now().add(1, "month").toDate();
```

I think `now()` and `now(new Date())` are the same

Reply:
yes